semgrep_output_v1.atd

```diff
@@ -1827,4 +1873,5 @@ type function_return
   | RetContributions of contributions
   | RetFormatter of string
   | RetValidate of bool
+  | RetResolveDependencies of resolve_dependencies_return
```

Comment:
Same, I don't think you need the extra intermediate record with just one field ...